app/dashboard/templates/index.html

```diff
@@ -139,6 +148,22 @@ <h4>&#8230;made easy!</h4>
             </ul>
         </div>
     </div>
+    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+        <div class="welcome-text">
+            <p>The KernelCI project is supported by the member organisations listed below:</p>
+            <div class="member-logos">
```

Comment:
@patersonc those style definitions aren't necessary, we already have some style definitions in `kernelci-frontend/app/dashboard/static/css/kci-base-2020.2.css` that could be used! For example: try `pull-center` instead of `member-logos`.